lib/buyers/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.conf.urls import patterns, url
+
+from rest_framework.routers import DefaultRouter
+
+from lib.buyers import views
+
+router = DefaultRouter()
+router.register(r'buyer', views.BuyerViewSet)
+
+urlpatterns = patterns(
+    '',
+    url(r'^confirm_pin', views.confirm_pin, name='confirm'),
+    url(r'^verify_pin', views.verify_pin, name='verify'),
+    url(r'^reset_confirm_pin', views.reset_confirm_pin,
+        name='reset_confirm_pin')
+) + router.urls
```

Comment:
The other code did this more like ``url(r'^', include(router.urls))`` which I think is more expclit